### PR TITLE
Add Atlassian to log regex

### PR DESCRIPTION
**Pull request by bseb**
### Background
New Log Type for Atlassian

### Changes
Added Atlassian to Log Regex

### Testing

Unit tests

### CODE REVIEW

File: panther_analysis_tool/schemas.py [hunk 1]
```diff
@@ -29,9 +29,10 @@
     r"\.Instance|Redshift\.Cluster|S3\.Bucket|WAF\.Regional\.WebACL|WAF\.WebACL)$"
 )
 LOG_TYPE_REGEX = Regex(
-    r"^(Apache\.AccessCombined|Apache\.AccessCommon|AWS\.ALB|AWS\.AuroraMySQLAudit|AWS"
```
Comment (bseb) [inline panther_analysis_tool/schemas.py:32]: Maybe we want to bump or ignore line length in this stanza? This is kinda painful to shuffle when trying to maintain alphabetical order

**Review by darwayne — APPROVED** [review]

File: panther_analysis_tool/schemas.py [hunk 2]
```diff
@@ -29,9 +29,10 @@
...
-    r"\.CloudTrail|AWS\.CloudTrailDigest|AWS\.CloudTrailInsight|AWS\.CloudWatchEvents|AWS"
-    r"\.GuardDuty|AWS\.S3ServerAccess|AWS\.VPCDns|AWS\.VPCFlow|AWS\.WAFWebACL|Box\.Event"
+    r"^(Apache\.AccessCombined|Apache\.AccessCommon|Atlassian\.Audit|AWS\.ALB"
```
Comment (nmartin413) [inline panther_analysis_tool/schemas.py:32]: should the `(` be unterminated?

Comment (bseb) [reply panther_analysis_tool/schemas.py:32]: It actually terminates on like 57. This is a wild regex.

Comment (bseb) [reply panther_analysis_tool/schemas.py:32]: It actually terminates on line 57.

**Review by nmartin413 — APPROVED** [review]

File: panther_analysis_tool/schemas.py (hunk 1 above)
Comment (lindsey-w) [reply panther_analysis_tool/schemas.py:32]: agreed, this is super annoying and hard to maintain

**Pull request merged**